### PR TITLE
Add missing tree types to prevent EvtGrow causing errors.

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Also adds a test to catch this in the future.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

**Review by Pikachu920 — APPROVED** [review]

**Comment by ShaneBeee** [comment]
while you’re in here, might as well update the missing trees?
https://github.com/SkriptLang/Skript/blob/dev/patch/src/main/java/ch/njol/skript/util/StructureType.java
https://github.com/SkriptLang/Skript/blob/49b2a377488ebec42ce7d1ed7d10cd0190f22237/src/main/resources/lang/default.lang#L280

**Comment by sovdeeth** [comment]
> while you’re in here, might as well update the missing trees? https://github.com/SkriptLang/Skript/blob/dev/patch/src/main/java/ch/njol/skript/util/StructureType.java
> 
> https://github.com/SkriptLang/Skript/blob/49b2a377488ebec42ce7d1ed7d10cd0190f22237/src/main/resources/lang/default.lang#L280

tbh didn't remember those existed lol

File: src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java [hunk 1]
```diff
@@ -312,6 +312,16 @@ public static boolean isAir(Material type) {
 		// cherry
 		if (Skript.isRunningMinecraft(1, 19, 4))
 			TREE_TO_SAPLING_MAP.put(TreeType.CHERRY, Material.CHERRY_SAPLING);
+
+		// mega pine (2x2 spruce tree with minimal leaves at top)
+		if (Skript.isRunningMinecraft(1, 20,5))
```
Comment (Efnilite) [inline src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java:317]:
```suggestion
		if (Skript.isRunningMinecraft(1, 20, 5))
```

**Review by Efnilite — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]

**Comment by APickledWalrus** [comment]
The StructureType class will be updated in a future PR.

**Pull request merged**